docs/Feature Documentation/App Service Provider Guide/index.md

```diff
@@ -209,6 +209,16 @@ Example SendLocation RPC Passing
 ![OnAppServiceData](https://raw.githubusercontent.com/smartdevicelink/sdl_evolution/master/assets/proposals/0167-app-services/rpc_passthrough.png)
 |||
 
+#### Embedded Navigation Best Practice
+
+The embedded navigation system on the IVI can be configured to be a navigation service provider. This feature is not limited to mobile applications.
```

Comment:
Might want a more generic statement that the IVI system can register any service type. Maybe this section could be under a `IVI App Service Providers` header overall and this statement could be included there.

Reply:
Good idea